aria/storage/structures.py

```diff
@@ -174,6 +181,12 @@ class IterPointerField(IterField, PointerField):
 
 
 class Model(object):
+    """
+    Base class for all of the storage models.
+    """
+    # Setting id to avoid missing class member is usage
+    id = None
```

Comment:
should take the comment off. if it makes sense for the "id" to be here then it makes sense, that's it.

where is it used in the base class anyway?